pkg/flink/types/materialized_results.go

```diff
@@ -158,6 +154,48 @@ func (s *MaterializedStatementResults) Append(rows ...StatementResultRow) bool {
 	return allValuesInserted
 }
 
+func (s *MaterializedStatementResults) processRowWithUpsertColumns(row StatementResultRow, rowKey string) {
```

Comment:
Can you refactor this so it's easier to understand where each flink operation is being handled? (+I, +U, -U, -D). Either some ifs/switch case or simply some comments

Reply:
done